src/main/java/redis/clients/jedis/gears/RedisGearsCommands.java

```diff
@@ -0,0 +1,23 @@
+package redis.clients.jedis.gears;
+
+import redis.clients.jedis.resps.GearsLibraryInfo;
+
+import java.util.List;
+
+public interface RedisGearsCommands {
+  String tFunctionLoad(String libraryCode);
+  String tFunctionLoad(String libraryCode, boolean replace);
```

Comment:
```suggestion
```
`config` is another optional argument. Ideally we should also add `tFunctionLoad(String libraryCode, String config);`.
Instead of having these many overloads, I'd prefer to remove this one.
We can (and should) keep this if we are confident that replace would be used way more than config (like 90% to 10% usage).

Reply:
Remove the overloads - only no args version and one with params